.github/workflows/regression.yml

```diff
@@ -456,6 +456,8 @@ jobs:
       - compute-metadata
       - upload-baseline-image-to-ecr
       - upload-comparison-image-to-ecr
+    env:
+      SINGLE_MACHINE_PERFORMANCE_API: ${{ secrets.SINGLE_MACHINE_PERFORMANCE_API }}
```

Comment:
As this is a protected workflow, any reason not to set this once at the top instead of in multiple jobs?

Reply:
I'll double check - my reading was that github doesn't support setting this above the `job` level.

Reply:
Here is an example where we do that for different env vars: https://github.com/vectordotdev/vector/blob/251c4c4608a70fd6c112ecacd0517c301f21e33c/.github/workflows/test.yml#L13-L28

Reply:
You're completely correct. I'll clean this up. Thanks, this will be a nice cleanup in the SMP repo as well.

Reply:
Done in [6e4f7c3](https://github.com/vectordotdev/vector/pull/17912/commits/6e4f7c3ff1c42cf2d8c74d73bb68838349005afd)